.clang-tidy

```diff
@@ -46,5 +46,5 @@ readability-container-size-empty,
 '
 HeaderFilterRegex: '^(c10/(?!test)|torch/csrc/).*$'
 AnalyzeTemporaryDtors: false
-WarningsAsErrors: '*'
+WarningsAsErrors: '*,-facebook-hte-*'
```

Comment:
Should we enable the clang-tidy check in open source as well (as misc-definitions-in-headers)?

Reply:
Yes we should!